test-data/unit/check-classes.test

```diff
@@ -8184,3 +8395,18 @@ class C:
     def f(self) -> None:
         __module__  # E: Name "__module__" is not defined
         __qualname__  # E: Name "__qualname__" is not defined
+
+[case testPropertySetterType]
+class A:
```

Comment:
Also check a generic class with a property that uses a type var type, and a subclass that uses non-trivial generic inheritance?

Reply:
There is `testOverridePropertyGeneric` for overrides, so I assume here you mean test _access_ on non-trivial subclass.